metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/CancelledSystemId.java

```diff
@@ -0,0 +1,11 @@
+package eu.europeana.metis.core.workflow;
+
+/**
+ * Enum that contains values used as identifiers for system specific cancellation operations.
+ *
+ * @author Simon Tzanakis (Simon.Tzanakis@europeana.eu)
+ * @since 2019-02-06
+ */
+public enum CancelledSystemId {
+  SYSTEM_MINUTE_CAP_EXPIRE
```

Comment:
Why put this in an enum? I don't think this adds any value. Why not create a string constant somewhere?

Reply:
I originally had it as a constant field in the class but then I needed in the unit tests so I found enum to be more clear for the time being.